src/groupREADME.md

```diff
@@ -0,0 +1,76 @@
+##Abstract:
+Many people have routine eating pallets and  what's cooking good looking offers many unique recipes as well as cooking instructions to advance a persons culinary experience. 
```

Comment:
You may want to capitalize "what's cooking good looking" here because it is the title of the application. There are also two spaces in front of what's.

Reply:
Saw your comments made the updates also fixed some other stuff for consistency. Great eye on that double spacing it was 100% not intended.

Reply:
Perfect! It looks great!